easybuild/easyconfigs/r/ReFrame/ReFrame-3.12.0.eb

```diff
@@ -0,0 +1,67 @@
+##
+# Author:    Robert Mijakovic <robert.mijakovic@lxp.lu>
+##
+easyblock = 'PythonBundle'
+
+name = 'ReFrame'
+version = '3.12.0'
+
+homepage = 'https://github.com/reframe-hpc/reframe'
+description = '''ReFrame is a framework for writing regression tests for HPC systems.'''
+
+toolchain = SYSTEM
+
+allow_system_deps = [('Python', SYS_PYTHON_VERSION)]
+
+req_py_majver = 3
+req_py_minver = 6
+
+use_pip = True
+
+exts_list = [
+    ('pip', '21.3.1', {
+        'use_pip': False,
+        'checksums': ['fd11ba3d0fdb4c07fbc5ecbba0b1b719809420f25038f8ee3cd913d3faa3033a'],
+    }),
+    ('wheel', '0.37.1', {
+        'source_tmpl': 'wheel-%(version)s-py2.py3-none-any.whl',
+        'checksums': ['4bdcd7d840138086126cd09254dc6195fb4fc6f01c050a1d7236f2630db1d22a'],
+    }),
+    ('setuptools', '60.0.3', {
```

Comment:
@akesandgren You should stick to an older `setuptools` version to avoid this:

```
ERROR: Package 'setuptools' requires a different Python: 3.6.8 not in '>=3.7'
```

See also https://github.com/easybuilders/easybuild-easyconfigs/blob/f590f2791d3dc5a26c7590ac11b794633e0cc18a/easybuild/easyconfigs/r/ReFrame/ReFrame-3.11.2.eb#L31

Reply:
If I do I get this:
```
uild_log.py:169 ERROR EasyBuild crashed with an error (at easybuild/base/exceptions.py:124 in __init__): Sanity check failed: extensions sanity check failed for 1 extensions: pip
failing sanity check for 'pip' extension: command "/usr/bin/python3 -c "import pip"" failed; output:
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 971, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 914, in _find_spec
  File "/home/a/ake/easybuild/software/ReFrame/3.12.0/lib/python3.8/site-packages/_distutils_hack/__init__.py", line 83, in find_spec
    return method()
  File "/home/a/ake/easybuild/software/ReFrame/3.12.0/lib/python3.8/site-packages/_distutils_hack/__init__.py", line 104, in spec_for_pip
    if self.pip_imported_during_build():
  File "/home/a/ake/easybuild/software/ReFrame/3.12.0/lib/python3.8/site-packages/_distutils_hack/__init__.py", line 115, in pip_imported_during_build
    return any(
  File "/home/a/ake/easybuild/software/ReFrame/3.12.0/lib/python3.8/site-packages/_distutils_hack/__init__.py", line 116, in <genexpr>
    frame.f_globals['__file__'].endswith('setup.py')
KeyError: '__file__',  (at easybuild/framework/easyblock.py:3472 in _sanity_check_step)
```

Reply:
See https://github.com/pypa/setuptools/issues/2941 for instance.